@throw(${1:exception});